scenario/src/main/kotlin/com/cosmotech/scenario/azure/ScenarioServiceImpl.kt

```diff
@@ -838,11 +840,13 @@ internal class ScenarioServiceImpl(
 
     when (eventBus.authentication.strategy) {
       SHARED_ACCESS_POLICY -> {
+        val (name, key) = kubernetesClient.getSecretFromKubernetes(eventHubNamespace, "phoenix")
```

Comment:
"phoenix" here should not be hardcoded
It could be stored in new app config entry

Reply:
For example: csm.platform.namespace